Source/WebCore/rendering/RenderThemeIOS.mm

```diff
@@ -1381,7 +1381,7 @@ static bool shouldUseConvexGradient(const Color& backgroundColor)
 
 Color RenderThemeIOS::autocorrectionReplacementMarkerColor(const RenderText& renderer) const
 {
-    auto caretColor = CaretBase::computeCaretColor(renderer.style(), renderer.textNode());
+    auto caretColor = CaretBase::computeCaretColor(renderer.style(), renderer.textNode(), false);
```

Comment:
This looks wrong. You probably want to use `renderer.page()`. Though once the new parameter goes away, that should just work.

Reply:
Specifically, I think the consequence the current implementation would be mismatched autocorrection underlines and caret color in Notes.